grafana-plugin/pkg/plugin/settings.go

```diff
@@ -317,32 +317,29 @@ func (a *App) SaveOnCallSettings(settings *OnCallPluginSettings) error {
 	return nil
 }
 
-func (a *App) GetSyncData(ctx context.Context, settings *OnCallPluginSettings) (*OnCallSync, error) {
+func (a *App) GetSyncData(pluginSettings *OnCallPluginSettings) (*OnCallSync, error) {
+	var err error
+
 	startGetSyncData := time.Now()
 	defer func() {
 		elapsed := time.Since(startGetSyncData)
 		log.DefaultLogger.Info("GetSyncData", "time", elapsed.Milliseconds())
 	}()
 
-	onCallPluginSettings, err := a.OnCallSettingsFromContext(ctx)
-	if err != nil {
-		return nil, fmt.Errorf("error getting settings from context = %v", err)
-	}
```

Comment:
I don't think we need to do:
```go
onCallPluginSettings, err := a.OnCallSettingsFromContext(ctx)
```
when `settings *OnCallPluginSettings` is already being passed into `GetSyncData` 🤔

Reply:
👍 good cleanup to make. This call was leftover from when there was some back and forth about what was coming from where.